addon/mixins/base.js

```diff
@@ -89,27 +88,35 @@ Semantic.BaseMixin = Ember.Mixin.create({
     };
   },
 
-  didInsertElement: function() {
+  initializeModule() {
     this.$()[this.get("module")](this.settings(this.get("module")));
+  },
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.initializeModule();
 
     var _this = this;
     var properties = {};
+
     // Modules without setable properties
-    if (['accordion', 'nag'].indexOf(this.get('module')) === -1) {
-      properties = this.execute('set');
-    }
+    properties = this.execute('internal', 'set');
     var property;
 
-    for(property in properties) {
-      if (!properties.hasOwnProperty(property)) {
-        continue;
-      }
+    if (typeof properties === "object" && !Ember.isArray(properties)) {
```

Comment:
Since this calls internal set now, when set doesn't doesn't exist the jquery module is returned. This allows us to check any module without blacklisting.